### ISSUE TITLE
Packer Default KMS-Key Error

### ISSUE THREAD

**Issue by rephric** [hashicorp/packer#7704]
- Packer version = 1.4.0
- Host platform = Mac

Hello Hashicorp,
I am currently trying to create an encrypted AMI. I am specifying the encrypt_boot option to true, and the kms key arn that I want it to be encrypted with.  It is defaulting to encryption with the default kms key of the AWS account, which is not allowing me to share the encrypted AMI across accounts.  Is this a known bug? Do I need to upgrade packer? 

"encrypt_boot": "true",
"kms_key_id": "arn:aws:kms:us-east-1:871234uybfnkrjenfjerfbhewjrbf,

**Comment by SwampDragons** [comment]
Hi!

Sorry this bug bit you.  We fixed it in https://github.com/hashicorp/packer/pull/7507 and it is included in the v1.4.1 release; if you upgrade, you'll hopefully see this bug go away. However, we also ended up with an AMI naming issue in the v1.4.1 release, which has been fixed in a branch but has not been merged yet-- binaries from that fix can be found [here](https://circleci.com/gh/hashicorp/packer/3166#artifacts/containers/0)
